Find a functionally equivalent code:android.graphics.ColorMatrix.setConcat:COMMENT
Method Modifier: public      
Comment:/**
 * Set this colormatrix to the concatenation of the two specified
 * colormatrices, such that the resulting colormatrix has the same effect
 * as applying matB and then applying matA. It is legal for either matA or
 * matB to be the same colormatrix as this.
 */

@@ -1,5 +1,5 @@
 {
-    float[] tmp = null;
+    float[] tmp;
     if (matA == this || matB == this) {
         tmp = new float[20];
     } else {

